src/ItemsContext.js

```diff
@@ -4,21 +4,35 @@ import firebase from 'firebase';
 export const ItemsContext = createContext();
 
 const ItemsProvider = props => {
-  //set useState([]) for list
   const [items, setItems] = useState([]);
   const db = firebase.firestore();
+  const userToken = localStorage.getItem('token');
 
   useEffect(() => {
     db.collection('items')
+      .where('token', '==', userToken)
       .get()
       .then(snapshot => {
         const list = snapshot.docs.map(doc => doc.data());
         setItems(list);
       });
-  }, []);
+  }, [db, userToken]);
```

Comment:
I found that the list is not updated when a user joins a non-empty list and is navigated to the `/list` route. Replacing `db` with `items` in the `useEffect` dependencies seems to solve the problem. This doesn't seem to be a problem introduce with this branch because it's happening in production too. I think think it's something we, or at least I, missed on reviewing last week.

Is the `db` dependency meant to be listening for changes in all documents in Firestore?

Reply:
The reason I added `[db, userToken]` was because my code editor was giving me an error when I left it blank `[]`. I saw that it works with `[items, userToken]`, but my code editor was still giving an error around it. I'm not exactly sure why, and I would love to discuss it during the meeting but having `[db, items, userToken]` seems to fix this issue. I just added the changes and pushed the code.